modules/eco-poison-pill-operator-troubleshooting.adoc

```diff
@@ -36,12 +36,22 @@ If the `MachineHealthCheck` controller did not create the `PoisonPillRemediation
 If the `PoisonPillRemediation` CR was created, ensure that its name matches the unhealthy node or the machine object.
 
 [id="daemon-set-exists_{context}"]
-== Daemon set exists even after uninstalling the Poison Pill Operator 
-Issue:: The Poison Pill daemon set exists even after after uninstalling the Operator.
+== Daemon set and other Poison Pill Operator resources exist even after uninstalling the Poison Pill Operator 
+Issue:: The Poison Pill Operator resources, such as the daemon set, configuration CR, and the remediation template CR, exist even after after uninstalling the Operator.
 
-Resolution:: To remove the Poison Pill daemon set, manually delete the `PoisonPillConfig` CR. Run the following command:  
+Resolution:: To remove the Poison Pill Operator resources, manually delete the resources by running the following commands for each resource type:
```

Comment:
Delete "manually." Since you're giving the commands, "manually" doesn't add anything.